kolibri/core/content/api.py

```diff
@@ -1387,6 +1388,7 @@ class Meta:
         fields = ("contentnode_id", "contentnode_id__in")
 
 
+@method_decorator(csrf_protect, name="dispatch")
```

Comment:
Good catch - I think we should actually just add the `IsAuthenticated` permissions class to this viewset, as it requires a logged in user in the `get_queryset` method, so would just give a 500 for an anonymous user, I'd guess?

This might be worth doing in 0.16 instead, if you could open a separate PR to add just that onto the release-v0.16.x branch.

Reply:
ok i will add IsAuthenticated permissions to this in a seperate PR
And will remove csrf changes from this